packages/manager/modules/pci/src/projects/project/storages/databases/database/acl/acl.controller.js

```diff
@@ -0,0 +1,35 @@
+export default class aclCtrl {
+  /* @ngInject */
+  constructor($translate, $q, CucCloudMessage) {
+    this.$translate = $translate;
+    this.$q = $q;
```

Comment:
Does not seem to be used

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1